crates/sui-protocol-config/src/lib.rs

```diff
@@ -3306,6 +3308,12 @@ impl ProtocolConfig {
                         cfg.feature_flags.accept_passkey_in_multisig = true;
                     }
                 }
+                77 => {
```

Comment:
FYI there is a 77 going out to mainnet via cherry pick from https://github.com/MystenLabs/sui/pull/21444 - you'll want to rebase after that lands on main and then move this to v78

Reply:
My intention is to enable this with the 2nd testnet release tomorrow 03/11. Isn't 77 going out both in testnet (tomorrow) and in mainnet (03/12 Wednesday)?

CC @ebmifa

Reply:
77 is in the release branch already, so you need to create v78 for this change